test/utils.cpp

```diff
@@ -119,6 +119,17 @@ TEST_CASE("Tokenizers behave correctly", "[utils]") {
 	}
 }
 
+TEST_CASE("utils::consolidate_whitespace replaces multiple consecutinve whitespace with a single space", "[utils]") {
```

Comment:
1. Sorry, I missed it last time: we no longer prefix function names with class names because we already have class name in the test's tag.
2. This line is overly long now; can you wrap it?